Gemfile

```diff
@@ -27,6 +27,7 @@ end
 gem "activerecord", "~> 4.1.11"
 gem "arel", "~> 5.0"
 gem "bcrypt-ruby", "~> 3.1.2"
+gem "clockwork", "~> 1.2.0"
```

Comment:
Can we lock this gem at the major version, see c3e41b4d1476f0d7bf9a45084ff5d979269b2ae9?